# Request
geometry_msgs/Point[] obstacle_centers
float64[] obstacle_radii
---
# Response
bool success
string message
